sklearn/neighbors/tests/test_nca.py

```diff
@@ -15,8 +15,8 @@
 from sklearn import clone
 from sklearn.exceptions import ConvergenceWarning
 from sklearn.utils import check_random_state
-from sklearn.utils._testing import (assert_raises,
-                                   assert_raise_message, assert_warns_message)
+from sklearn.utils._testing import (assert_raise_message,
```

Comment:
Could you do the same pattern change with `assert_raise_message` as well.
We are now using the same matching with `pytest.raises`.

Reply:
I will work in this.